tests/unit/helpers/eventsRepetition.test.js

```diff
@@ -372,7 +376,33 @@ describe('createRepetitionsEveryDay', () => {
       moment().range(moment('2019-01-11T09:00:00.000Z'), moment('2019-04-10T09:00:00.000Z')).by('days')
     );
 
-    await EventsRepetitionHelper.createRepetitionsEveryDay(event, sector, range);
+    dayDiff.returns(0);
+
+    await EventsRepetitionHelper.createRepetitionsEveryDay(event, sector);
+
+    sinon.assert.calledOnceWithExactly(
+      createRepeatedEvents,
+      event,
+      sinon.match(calledRange => JSON.stringify(calledRange) === JSON.stringify(range)),
+      sector,
+      false
+    );
+  });
+
+  it('should create repetition every day from first event to current time plus 90 days #tag', async () => {
```

Comment:
should create repetition every day with first event in the past (+ retirer tag)

Reply:
done